cmd/prom-aggregation-gateway/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM alpine:3.10
+FROM alpine:3.14.1
```

Comment:
Any reason for changing this?

Reply:
Just a good security practice in that case where we just put a single binary. (tested successfully)